client/signup/steps/user/index.jsx

```diff
@@ -102,6 +102,11 @@ export class UserStep extends Component {
 	};
 
 	componentDidUpdate() {
+		if ( this.props.step?.status === 'completed' ) {
```

Comment:
I wonder why this wasn't here before! Is this what was causing the one step to flash for a second during the flow?

Reply:
It was 👍